### DISCUSSION TITLE
Difference between serve and build structure (unwanted folder listing)

### DISCUSSION THREAD

**Discussion by robozb** [mkdocs/mkdocs#3263] [Q&A] [answered]
I don't want mkdocs to list the subfolders, just showing the subpages. How can I achieve this requirement? This works well in "mkdocs serve" but not in the "mkdocs build". 

Thanks: Bela

![image](https://github.com/mkdocs/mkdocs/assets/8656998/762d4736-6ad4-4040-8439-6f26595d1926)

Unwanted behavior:

https://github.com/mkdocs/mkdocs/assets/8656998/e3b18cbb-1ffb-40f3-89a9-91051736833f

Perfect functionality: 

https://github.com/mkdocs/mkdocs/assets/8656998/648a4372-21b2-4de5-ad43-6d91e6181a4a

**Comment by pawamoy** [comment]
Resolved on Gitter. User was opening the built site's index.html within the browser, without `python -m http.server` or `mkdocs serve` (or any other server). Site is built with directory URLs.

**Comment by robozb** [comment]
Duplicate of this: https://github.com/mkdocs/mkdocs/issues/926

Thank you so much dear @pawamoy

**Comment by robozb** [comment, marked as answer]
The solution: 
in mkdocs.yml file:
---------------------
use_directory_urls: false

and then: mkdocs build
